tools/apiview/parsers/js-api-parser/src/jstokens.ts

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import jsTokens from "js-tokens";
+import { type ReviewToken, TokenKind } from "./models";
+
+const TS_KEYWORDS = new Set<string>([
```

Comment:
Where did this list come from and what does it impact? Just colorization? Or something else

Reply:
I should link to it in a comment https://github.com/microsoft/TypeScript/blob/aa9df4d68795052d1681ac7dc5f66d6362c3f3cb/src/compiler/scanner.ts#L135

Reply:
yes I think so, colorization for keywords in apiview.